### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by figurative666** [open-mmlab/mmpretrain#1657]
### Branch

main branch (mmpretrain version)

### Describe the bug

Hello, why is paddle training better than mmclass framework when the same dataset has been used? My accuracy on paddle is 0.9871, but on mmclass it is 22.4299
使用的指令 python tools/train.py configs/mobilenet_v3/mobilenet_v3_small_imagenet.py 
数据集格式

### Environment

{'sys.platform': 'win32',
 'Python': '3.8.16 (default, Mar  2 2023, 03:18:16) [MSC v.1916 64 bit '
           '(AMD64)]',
 'CUDA available': True,
 'numpy_random_seed': 2147483648,
 'GPU 0': 'NVIDIA GeForce GTX 1050 Ti',
 'CUDA_HOME': 'C:\\Program Files\\NVIDIA GPU Computing Toolkit\\CUDA\\v11.6',
 'NVCC': 'Cuda compilation tools, release 11.6, V11.6.55',
 'MSVC': '用于 x64 的 Microsoft (R) C/C++ 优化编译器 19.35.32216.1 版',
 'GCC': 'n/a',
 'PyTorch': '1.10.1',
 'TorchVision': '0.11.2',
 'OpenCV': '4.7.0',
 'MMEngine': '0.7.4',
 'MMCV': '2.0.0',
 'MMPreTrain': '1.0.0rc8+09a3fe7'}

**Comment by Ezra-Yu** [comment]
I think there must be something wrong with your config. Can you show me your config?
